backend/src/main/java/de/laura/project/controller/ApiController.java

```diff
@@ -0,0 +1,30 @@
+package de.laura.project.controller;
+
+import de.laura.project.api.model.TriviaApiData;
+import de.laura.project.model.TriviaDataPoint;
+import de.laura.project.service.TriviaService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("questions")
+
+public class ApiController {
```

Comment:
I would maybe pick a more suitable to reflect the job of the controller, e.g. TriviaController or QuestionsController

Reply:
Thanks!